lib/resources/nginx.rb

```diff
@@ -13,20 +13,22 @@ class Nginx < Inspec.resource(1)
       describe nginx do
         its('conf_path') { should cmp '/etc/nginx/nginx.conf' }
       end
-      describe nginx('/etc/sbin') do
+      describe nginx('/etc/sbin/') do
         its('version') { should be >= '1.0.0' }
       end
+      describe nginx do
+        its('modules') { should include 'my_module' }
+      end
     "
     attr_reader :params, :bin_dir
 
-    def initialize(nginx_bin_dir = nil)
+    def initialize(path = nil)
       return skip_resource 'The `nginx` resource is not yet available on your OS.' if inspec.os.windows?
-      bin_dir = nginx_bin_dir || bin_dir_from_command || locate_by_path('bin')
-      bin_dir.chop! if bin_dir.end_with? '/'
-      if bin_dir.nil? || !inspec.command("#{bin_dir}/nginx").exist?
-        return skip_resource 'Error fining the NGINX binary or `bin_dir`, plese review the provided `PATH` is only the `base_dir` and or ensure the `nginx` binary is in the system `PATH`.'
-      end
-      cmd = inspec.command("#{bin_dir}/nginx -V 2>&1")
+      path ||= '/usr/sbin/'
```

Comment:
Let's be consistent with other resources... rather than taking a directory and then joining it to `nginx`, the user should supply a full path to the nginx binary.

```ruby
def initialize(nginx_path = '/usr/sbin/nginx')
  return skip_resource 'blah blah..."
end
```

Reply:
Done. Thank you @adamleff.